new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -731,6 +737,162 @@ public void testCreateNewCluster() {
+
+    //verify the pods are restarted
+    verifyRollingRestartOccurred(pods, 1, introDomainNamespace);
+
+    // verify the admin server service created
+    checkServiceExists(adminServerPodName, introDomainNamespace);
+
+    // verify admin server pod is ready
+    checkPodReady(adminServerPodName, domainUid, introDomainNamespace);
+
+    // verify new cluster managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Checking managed server service {0} is created in namespace {1}",
+          managedServerPodNamePrefix + i, introDomainNamespace);
+      checkServiceExists(managedServerPodNamePrefix + i, introDomainNamespace);
+    }
+
+    // verify new cluster managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Waiting for managed server pod {0} to be ready in namespace {1}",
+          managedServerPodNamePrefix + i, introDomainNamespace);
+      checkPodReady(managedServerPodNamePrefix + i, domainUid, introDomainNamespace);
+    }
+
+    logger.info("Getting the list of servers using the listServers");
+    String baseUri = "http://" + K8S_NODEPORT_HOST + ":" + adminServerT3Port + "/clusterview/";
+    String serverListUri = "ClusterViewServlet?listServers=true";
+    HttpResponse<String> response = null;
+    for (int i = 0; i < 5; i++) {
+      assertDoesNotThrow(() -> TimeUnit.SECONDS.sleep(10));
+      response = assertDoesNotThrow(() -> OracleHttpClient.get(baseUri + serverListUri, true));
+      assertEquals(200, response.statusCode(), "Status code not equals to 200");
+    }
+
+    // verify managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Checking {0} health", managedServerNameBase + i);
+      assertTrue(response.body().contains(managedServerNameBase + i + ":HEALTH_OK"),
+          "Didn't get " + managedServerNameBase + i + ":HEALTH_OK");
+    }
+
+    logger.info("Getting node port for default channel");
+    int serviceNodePort = assertDoesNotThrow(() -> getServiceNodePort(
+        introDomainNamespace, adminServerPodName + "-external", "default"),
+        "Getting admin server node port failed");
+
+    logger.info("Validating WebLogic admin server access by login to console");
+    boolean loginSuccessful = assertDoesNotThrow(()
+        -> adminNodePortAccessible(serviceNodePort, ADMIN_USERNAME_PATCH, ADMIN_PASSWORD_PATCH),
+        "Access to admin server node port failed");
+    assertTrue(loginSuccessful, "Console login validation failed");
+
+    logger.info("Validating WebLogic admin server access by login to console using old credentials");
+    assertThrows(AssertionFailedError.class, ()
+        -> adminNodePortAccessible(serviceNodePort, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT),
+        "Accessing using old user/password succedded, supposed to fail");
+    assertTrue(loginSuccessful, "Console login validation failed");
```

Comment:
fix the assert and loginSuccessful is not captured

Reply:
the assert only to verify that the assert is thrown. removed the assertTrue(loginSuccessful, "Console login validation failed") line